department-of-veterans-affairs/caseflow
# Update Storybook to Work on Demo

Connects #14022

### Description
This changes the target directory for Storybook build output, placing it under `/public` and hence making it available for viewing via normal routing.

### Acceptance Criteria
- [ ] Storybook assets are compiled and output to `public/storybook`
- [ ] Storybook can be accessed at `/storybook` locally (if built) and on demo

### Testing Plan — Local
1. Run `yarn storybook-build` from `client` folder
2. Confirm assets are built to `public/storybook`
3. Access [http://localhost:3000/storybook/](http://localhost:3000/storybook/)
